package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "wallet-web-ui-b",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
```

Comment:
You have a type declaration file for Jest, but i cannot see any unit test in the codebase, kindly remove it if you are not using it